### PR TITLE
opentelemetry collector documentation on monitoring jobs

### PR DESCRIPTION
**Pull request by c-kruse**
## Purpose of this pull request

This pull request adds documentation introducing monitoring jobs, a new feature in the sumo opentelemetry collector.

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [X] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

**Comment by jpipkin1** [comment]
@c-kruse, I noticed this pull request is in draft. When you are ready for us to review it, click the "Ready for review" button.

File: docs/send-data/opentelemetry-collector/data-source-configurations/collect-logs.md [hunk 1]
```diff
@@ -538,10 +539,50 @@ If you can see the following logs from the collector after applying the configur
 
 Make sure you are using collector version `v0.78.0-sumo-0` or higher.
 
+## Collecting logs from other sources
+
+The full list of receivers available can be found on [Sumo Logic OpenTelemetry Collector page][collector_components_docs]. When the Sumo Logic OpenTelemetry Collector does not support collecting logs from a source, the [Monitoring Job Receiver (Beta)][monitoring_job_receiver_docs] can be used as a catch-all.
```
Comment (kimsauce) [inline docs/send-data/opentelemetry-collector/data-source-configurations/collect-logs.md:544]:
```suggestion
You can find the full list of receivers available on the [Sumo Logic OpenTelemetry Collector][collector_components_docs] page. When the Sumo Logic OpenTelemetry Collector does not support collecting logs from a source, the [Monitoring Job Receiver (Beta)][monitoring_job_receiver_docs] can be used as a catch-all.
```

File: docs/send-data/opentelemetry-collector/data-source-configurations/collect-logs.md [hunk 2]
```diff
@@ -538,10 +539,50 @@ If you can see the following logs from the collector after applying the configur
 
 Make sure you are using collector version `v0.78.0-sumo-0` or higher.
 
+## Collecting logs from other sources
+
+You can find the full list of receivers available on the [Sumo Logic OpenTelemetry Collector][collector_components_docs] page. When the Sumo Logic OpenTelemetry Collector does not support collecting logs from a source, the [Monitoring Job Receiver (Beta)][monitoring_job_receiver_docs] can be used as a catch-all.
+
+The Monitoring Job Receiver can be configured to execute a script or command on the collector host. The standard output and error streams from that command will be collected as log record(s).
+
+:::important  Prefer purpose-built receivers to Monitoring Jobs.
```
Comment (kimsauce) [inline docs/send-data/opentelemetry-collector/data-source-configurations/collect-logs.md:548]:
```suggestion
:::important Prefer purpose-built receivers to Monitoring Jobs
```

File: docs/send-data/opentelemetry-collector/data-source-configurations/collect-logs.md [hunk 3]
```diff
@@ -538,10 +539,50 @@ If you can see the following logs from the collector after applying the configur
...
+
+You can find the full list of receivers available on the [Sumo Logic OpenTelemetry Collector][collector_components_docs] page. When the Sumo Logic OpenTelemetry Collector does not support collecting logs from a source, the [Monitoring Job Receiver (Beta)][monitoring_job_receiver_docs] can be used as a catch-all.
+
+The Monitoring Job Receiver can be configured to execute a script or command on the collector host. The standard output and error streams from that command will be collected as log record(s).
+
+:::important Prefer purpose-built receivers to Monitoring Jobs
+When available, prefer purpose-built receivers like `sqlquery` to monitoring jobs receiver. This example monitoring job configuration runs SQL queries using the `psql` tool for demonstration purposes as it is well understood.
+:::
+
+This configuration configures the controller to execute the `psql` command and collects the output text as logs. See the [monitoring job documentation][monitoring_job_receiver_docs] for configuration specifics including advanced output processing with features like multi-line log detection, timestamp and field parsing.
```
Comment (kimsauce) [inline docs/send-data/opentelemetry-collector/data-source-configurations/collect-logs.md:552]:
```suggestion
This configuration configures the controller to execute the `psql` command and collects the output text as logs. See the [Monitoring Job Receiver documentation][monitoring_job_receiver_docs] for configuration specifics including advanced output processing with features like multi-line log detection, timestamp and field parsing.
```

**Review by kimsauce — APPROVED** [review]

**Pull request merged**